code/game/machinery/camera/camera.dm

```diff
@@ -241,6 +241,10 @@ var/list/camera_names=list()
 		..()
 	return
 
+/obj/machinery/camera/attack_ai(mob/user as mob)
+	if(istype(user,/mob/living/silicon/pai))

```

Comment:
It doesn't work, I commented out | WIREJACK

It should ideally, so you can jump from camera to camera, but it just doesn't work right now.